src/state/governance/hooks.ts

```diff
@@ -11,6 +11,7 @@ import { TransactionResponse } from '@ethersproject/providers'
 import { useTransactionAdder } from '../transactions/hooks'
 import { useState, useEffect, useCallback, useMemo } from 'react'
 import { abi as GOV_ABI } from '@uniswap/governance/build/GovernorAlpha.json'
+import { t } from '@lingui/macro'
```

Comment:
is the use of `t` discouraged @justindomingue ? remember seeing something about this

Reply:
`t` doesn't reload on hooks (locale) change. `<Trans>` is preferred for that reason

https://github.com/lingui/js-lingui/issues/1090